test/jdk/java/lang/RuntimeTests/exec/ExecCommand.java

```diff
@@ -223,27 +221,11 @@ void testCommandAmbiguousEmpty(String command, String testFile, List<Class<Excep
     void testCommandAmbiguousFalse(String command, String testFile, List<Class<Exception>> perModeExpected) {
         Properties props = System.getProperties();
         props.setProperty(JDK_LANG_PROCESS_ALLOW_AMBIGUOUS_COMMANDS, "false");
-        System.setSecurityManager(null);
 
         testCommandMode(command, "Ambiguous false", testFile,
                 perModeExpected.get(Mode.FALSE_NO_SM.ordinal()));
     }
 
-    /**
-     * Test each command with SecurityManager and default allowAmbiguousCommands is empty.
-     * @param command a command
-     * @param perModeExpected an expected Exception class or null
-     */
-    @Test(dataProvider = "TEST_RTE_ARGS")
-    void testCommandWithSM(String command, String testFile, List<Class<Exception>> perModeExpected) {
-        Properties props = System.getProperties();
-        props.setProperty(JDK_LANG_PROCESS_ALLOW_AMBIGUOUS_COMMANDS, "");
-        System.setSecurityManager(new SecurityMan());
```

Comment:
I assume SecurityMan should be removed too.

Reply:
Yes, and the comments that say "no SM" should be updated. @RogerRiggs can you take a look at this? Thanks.

Reply:
Fix to ExecCommand pushed to the sandbox.